.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
external_sst_file_test was actually the test that motivated this patch in the first place. I already checked with @IslamAbdelRahman and he was ok with moving it to MEM_ENV. What criteria you have in mind when you include/exclude tests from MEM_ENV?